spec/unit/views/components/paginated_collection_spec.rb

```diff
@@ -91,6 +91,32 @@ def paginated_collection(*args)
       end
     end
 
+    context "when specifying download_url_options" do
+      let(:view) do
+        view = mock_action_view
+        allow(view.request).to receive(:query_parameters).and_return details: '1'
```

Comment:
Line is too long. [81/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.